azure-pipelines.yml

```diff
@@ -442,14 +442,47 @@ stages:
               vs2022_latest89:
                 PRODUCT: "SonarQube"
                 SQ_VERSION: "LATEST_RELEASE[8.9]"
-                SONAR_CFAMILYPLUGIN_VERSION: "6.20.5.49286"  # LATEST_RELEASE of CFAMILY is not compatible with old SQ
+                SONAR_CFAMILYPLUGIN_VERSION: "6.20.5.49286"  # LATEST_RELEASE of CFAMILY is not compatible with SQ 8.9
```

Comment:
I don't remember if this is true exactly as written. It's for sure not compatible with 7.9, I'm not sure about 8.9. Probably yes.
Also "old SQ" is good enough to avoid maintaining that number on 2 lines.
I'd revert this.

Reply:
Please also update the new lines to the same text, for simplicity